AlayaCare Cloud Tech Radar.csv

```diff
@@ -9,35 +9,39 @@
+"Adopt","CircleCI","Platforms","FALSE","<p><a href='https://circleci.com/'>circleci.com</a></p>"
 "Adopt","PostGres","Platforms","FALSE","<p></p>"
+"Adopt","MySQL","Platforms","FALSE","<p></p>"
+"Adopt","Microsoft Visio","Tools","TRUE","<p>Recommended for individual work when then end result requires polishing, e.g. C4 architecture diagrams.</p>"
+"Adopt","Draw.io","Tools","TRUE","<p>Recommended for individual work when then end result requires polishing, e.g. C4 architecture diagrams.</p>"
+"Adopt","FigJam","Tools","TRUE","<p>Rich whiteboard tool that supports advanced use cases.</p>"
 "Adopt","Renovate","Tools","TRUE","<p><a href='https://www.npmjs.com/package/renovate'>Renovate</a> - Save time and reduce risk by automating dependency updates in software projects.</a></p>"
 "Adopt","SCSS","languages-and-frameworks","FALSE","<p></p>"
-"Adopt","SonarCloud","Platforms","TRUE","<p><a href='https://sonarcloud.io/'>SonarCloud</a>’s static analysis detects Bugs and Code Smells in your repositories and provides the feedback you need to write better code.</a></p>"
+"Adopt","SonarCloud","Platforms","FALSE","<p><a href='https://sonarcloud.io/'>SonarCloud</a>’s static analysis detects Bugs and Code Smells in your repositories and provides the feedback you need to write better code.</a></p>"
 "Adopt","Vue CLI","languages-and-frameworks","FALSE","<p><a href='https://cli.vuejs.org/'>Vue CLI</a> offers standard tooling for Vue.js development.</p>"
 "Adopt","Vue Test Utils","languages-and-frameworks","FALSE","<p><a href='https://vue-test-utils.vuejs.org/'>Vue Test Utils</a> is the official unit testing utility library for Vue.js.</p>"
 "Adopt","Webpack","languages-and-frameworks","FALSE","<p><a href='https://webpack.js.org/'>Webpack</a></p>"
 "Adopt","Python 3","languages-and-frameworks","FALSE","<p></p>"
-"Adopt","CircleCI","Platforms","FALSE","<p><a href='https://circleci.com/'>circleci.com</a></p>"
+"Adopt","Vue 3","languages-and-frameworks","TRUE","<p>Vue 3 is the prefered language for new front-end developement.</p>"
+"Adopt","TypeScript","languages-and-frameworks","TRUE","<p>TypeScript is the prefered language for new front-end developement.</p>"
+"Adopt","Micro frontend","Techniques","FALSE","<p></p>"
+"Adopt","Backstage","Platforms","FALSE","<p>Backstage is the source of truth for our component's scorecards</p>"
 "Trial","GitOps","Techniques","FALSE","<p>GitOps is a way of implementing Continuous Deployment for cloud native applications. It focuses on a developer-centric experience when operating infrastructure, by using tools developers are already familiar with, including Git and Continuous Deployment tools.</p>"
-"Trial","Micro frontend","Techniques","FALSE","<p></p>"
-"Trial","Snyk.io","Platforms","TRUE","<p><a href='https://snyk.io/'>snyk.io</> - Automatically detect vulnerabilities</a></p>"
+"Trial","Snyk.io","Platforms","FALSE","<p><a href='https://snyk.io/'>snyk.io</> - Automatically detect vulnerabilities</a></p>"
```

Comment:
At this time and until further notice Snyk.io would be **Adopt**, because it's part of the Security Scorecard. It's not used to its full power and purpose, but it's part of our processes and swapping it for another tool would require updating the scoring procedure and scorecard definition https://alayacare.atlassian.net/wiki/spaces/development/pages/3607199745/Component+Scorecard+-+Lagging+security+vulnerabilities+or+OSS+license+issues